readme_renderer/markdown.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2014 Donald Stufft
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import, division, print_function
+
+from CommonMark import commonmark
+
+from .clean import clean
+
+
+variants = {
+    "CommonMark": commonmark,
+}
+
+
+def render(raw, variant="CommonMark"):
+    renderer = variants.get(variant, commonmark)
```

Comment:
Does it make sense to fall back to CommonMark here if the variant is not known? I'm kind of thinking that it might be better to treat that as an error case, and let callers handle the fallback. Otherwise it isn't possible to determine a variant that doesn't exist.

Reply:
Good call. Addressed.